packages/desktop-client/src/components/reports/spreadsheets/default-spreadsheet.tsx

```diff
@@ -282,8 +283,11 @@ export default function createSpreadsheet(
       };
     });
 
-    const categoryGroupCalcData = catGroup.map(group => {
-      if (hidden || group.hidden === 0) {
+    const categoryGroupCalcData = catGroup
+      .filter(f => {
+        return (hidden || f.hidden === 0) && f;
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏why do you need to check `&& f`? If a few chars before you're checking the value of it (`f.hidden`). Presumably it will always evaluate as `true`?